### ISSUE TITLE
Test never stops running.

### ISSUE THREAD

**Issue by Jarmo-Baltzar** [rspieker/jest-transform-svelte#18]
Hi , Noticed there was issue that is closed, but  i have the same issue now.  It just keeps on running. Latest version of jest-transform-svelte.

**Comment by beomy** [comment]
I have same issue too. I use 12.7.0 Node.js version.
I think this issue reason is deasync at lastest node version

**Comment by RobbieTheWagner** [comment]
Yes, we are also experiencing this on https://github.com/shipshapecode/shepherd
